### ISSUE TITLE
Profiling noise when library loaded

### ISSUE THREAD

**Issue by aepstein** [nodegit/nodegit#885]
I started using nodegit 0.9.0 and it seems to work fine, except I get the following noise at the end of every test run (using mocha).  Could this be an issue with binary library it pulled in?  Same happens in 0.8.0.  If I do not have a `require('nodegit')`, none of this output happens.  This seems to happen on exit.  If I run node shell and require the library, the same messages appear when I close it with `process.exit();`

```
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/writestream.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/treebuilder.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/tree_entry.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/tree.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/transport.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/transfer_progress.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/transaction.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/trace.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/time.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/tag.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/submodule_update_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/submodule.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/strarray.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/status_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/status_list.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/status_entry.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/status.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/stash_apply_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/stash.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/smart.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/signature.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/revwalk.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/revparse.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/revert_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/revert.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/reset.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/repository_init_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/repository.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/remote_callbacks.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/remote.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/refspec.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/reflog_entry.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/reflog.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/reference.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/refdb.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/rebase_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/rebase_operation.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/rebase.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/push_update.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/push_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/push.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/pathspec_match_list.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/pathspec.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/patch.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/packbuilder.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/openssl.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/oidarray.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/oid_shorten.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/oid.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/odb_object.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/odb.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/object.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/note_iterator.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/note.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/message.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/merge_result.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/merge_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/merge_file_result.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/merge_file_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/merge_file_input.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/merge.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/mempack.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/libgit2.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/indexer.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/index_time.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/index_entry.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/index_conflict_iterator.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/index.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/ignore.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/hashsig.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/graph.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/giterr.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/filter_list.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/filter.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/fetch_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/fetch.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/error.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_stats.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_perfdata.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_line.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_hunk.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_find_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_file.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_delta.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_binary_file.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff_binary.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/diff.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/describe_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/describe_format_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cvar_map.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cred_userpass_payload.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cred_username.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cred_default.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cred.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/config_entry.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/config.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/commit.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/clone_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/clone.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cherrypick_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cherrypick.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/checkout_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/checkout.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cert_x509.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cert_hostkey.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/cert.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/buf.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/branch_iterator.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/branch.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/blob.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/blame_options.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/blame_hunk.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/blame.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/attr.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/annotated_commit.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/str_array_converter.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/functions/sleep_for_ms.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/functions/copy.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/wrapper.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/promise_completion.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/nodegit.gcda:Skip
profiling:/home/travis:Cannot create directory
profiling:/home/travis/build/nodegit/nodegit/build/Release/obj.target/nodegit/src/lock_master.gcda:Skip
```

**Comment by tbranyen** [comment]
Yup, I've commented on this as well. We took a stab at eliminating this noise, but I guess I just don't understand $TRAVIS_TAG well enough.

For reference: https://github.com/nodegit/nodegit/commit/bac54d2f17e53245971c06a529781661959320b5

**Comment by tbranyen** [comment]
@aepstein I'll look more into this tomorrow FYI

**Comment by tbranyen** [comment]
Fixed in 0.11!
